dynamiq/nodes/node.py

```diff
@@ -239,6 +239,8 @@ class Node(BaseModel, Runnable, ABC):
     input_schema: type[BaseModel] | None = None
     callbacks: list[NodeCallbackHandler] = []
     _json_schema_fields: ClassVar[list[str]] = []
+    clone_reset_methods: ClassVar[list[str]] = ["reset_run_state"]
+    clone_reset_attr_initializers: ClassVar[dict[str, Callable[["Node"], Any]]] = {}
```

Comment:
let's use more common naming as it could include some of post-init attributes.
Let's skip for now Callable initializers list (but leave method where you define logic with callables) as I don't know how to manage any dump/load of them:
```suggestion
    _clone_init_methods_names: ClassVar[list[str]] = ["reset_run_state"]
```

Please update naming of helpers. and check them